setup.py

```diff
@@ -17,12 +17,15 @@
     keywords=['NLP', 'transfer learning', 'language models', 'NLU'],
```

Comment:
can we also bump the version above on line 8? not sure why i can't comment on that line.